app/src/main/kotlin/com/vrem/wifianalyzer/wifi/model/WiFiSignal.kt

```diff
@@ -73,7 +74,24 @@ data class WiFiSignal(
     companion object {
         const val FREQUENCY_UNITS = "MHz"
 
+        const val RM_ENABLED_CAPABILITIES_IE = 70
+        const val NEIGHBOR_REPORT_IDX = 0
+        const val NEIGHBOR_REPORT_BIT = 1
+
+        const val EXTENDED_CAPABILITIES_IE = 127
+        const val BSS_TRANSITION_IDX = 2
+        const val BSS_TRANSITION_BIT = 3
+
+        const val MOBILE_DOMAIN_IE = 54
         val EMPTY = WiFiSignal()
     }
 
-}
\ No newline at end of file
+}
+
+enum class FastRoaming(val protocol :String) {
```

Comment:
Recommendation:
- To create separate class file for `enum` `FastRoaming `
- To move transformation logic from `Transformer` into this class

Reply:
done